.github/workflow-templates/test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         mkdir -p $GOPATH/src/github.com/ovn-org
         mv $GITHUB_WORKSPACE $GOPATH/src/github.com/ovn-org/ovn-kubernetes
         echo "::set-env name=WORKDIR::$GOPATH/src/github.com/ovn-org/ovn-kubernetes"
+        mkdir -p $GITHUB_WORKSPACE
```

Comment:
@as-com @trozet at L32 we are doing a `mv`, but at L34 we are trying to create this directory? Am I missing something?

Reply:
We recreate this directory because some steps assume $GITHUB_WORKSPACE always exists.